src/main/java/model/input/file/PlainTextFile.java

```diff
@@ -42,7 +42,7 @@ public DataTable createDataTable() throws IOException {
 			skipToStartLine(scanner);
 			if (hasFirstRowAsHeader()) {
 				String headers = scanner.nextLine();
-				String[] sections = headers.split(delimiter);
+				String[] sections = headers.split(",");
```

Comment:
de delimiter moet worden opgegeven in de xml file, dus hier moet juist een variabele staan en die variabele moet worden geset door de wizard.